cypress/features/regression/portrait.feature

```diff
@@ -42,12 +39,8 @@ Feature: Portrait default component
 
 	@positive
 	Scenario Outline: Set Portrait src to <source>
-		Given I select source to "src"
-		When I set src to "<source>"
+		When I open default "Portrait" component in noIFrame with "portrait" json from "commonComponents" using "src" object name
 		Then Portrait src value is set to "<source>"
```

Comment:
how you are passing this `source` value here ?

Reply:
fixed